### PR TITLE
Fix #304

### LINKED ISSUE #304: first item in a level (level: 1.0) gets formatted differently from the rest (level: 1.1, 1.2, etc.) when `door publish`
Is this intentional, that when `door add REQ` three times, then `door publish REQ doc.html` or `door publish REQ doc.md`

The first item is one level up than the rest. 

It can be traced back to the `def _format_level(level):` function inside `publisher.py` where

    def _format_level(level):
    """Convert a level to a string and keep zeros if not a top level."""
    text = str(level)
    if text.endswith('.0') and len(text) > 3:
        text = text[:-2]
    return text

So `level: 1.0` the ending zero was thrown out. 

Is there a reason for that? Prettier document or a traditional style?

### PR DESCRIPTION
**Pull request by RomainTT**
Update description of headings to avoid confusion.

### CODE REVIEW

**Comment by RomainTT** [comment]
Fix #304 (just a comment for github automation)

File: docs/reference/items.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ text: |
 When this item is published, Doorstop will create a new heading with the text
 "1.1.0 This is the heading" and put the remaining text into its body.
 
-#### Example: Normative item
+#### Example: Normative heading
```
Comment (jacebrowning) [inline docs/reference/items.md]: I believe this will only be rendered as a heading if there are items nested under it, correct?

Comment (RomainTT) [reply docs/reference/items.md]:
```shell
doorstop create SRS srs
doorstop add SRS
doorstop publish SRS one_item.md
doorstop add SRS
doorstop add SRS
doorstop publish SRS three_items.md
```

Result in *one_item.md*:

```
# 1.0 SRS001 {#SRS001 }
```

Result in *three_items.md*:

```
# 1.0 SRS001 {#SRS001 }

## 1.1 SRS002 {#SRS002 }

## 1.2 SRS003 {#SRS003 }

```

It seems it is always rendered as a heading. Or am I missing something?

Comment (jacebrowning) [reply docs/reference/items.md]: Well, they're all technically "headings" in Markdown. 😄 

The more that I think about this, I think that "Normative heading" will cause more confusion. I'm inclined to close #304 as "working as intended". I would appreciate a cleanup PR that fixes the triple backticks.

**Pull request closed without merging**